container/chewbbaca

```diff
@@ -1,32 +1,21 @@
 Bootstrap:docker
-From:python:3.7.11
+From:condaforge/mambaforge:latest
```

Comment:
Always a bit dangerous to use :latest
Better to spec a version even though it _should not_ break, it might

Reply:
Good point. Will fix that!